admin2.php

```diff
@@ -22,46 +22,86 @@
 	</head>
 	
 	<body>
-		
+	
 		<!-- grid-container class which contains all content using CSS grid -->
 		<div class="grid-container">
 			
-			<!-- navigation located at the very top -->				
+			<!-- navigation located at the very top -->	
 			<div class="nav">
 				<b><p style = "color: white; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Navigation</p></b>
 			</div>
+			
+			<!-- login box -->
+			<div class = "login_box">

```

Comment:
Moved login_box from being contained inside content_1 to being just contained inside grid-container.